region/maridia/outer/Mama Turtle Room.json

```diff
@@ -955,44 +1020,88 @@
       "requires": [
         "canPreciseGrapple",
         "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Grapple, Mama Turtle is Sleeping",
+      "requires": [
+        "canUseGrapple",
+        {"obstaclesNotCleared": ["A"]}
       ],
-      "devNote": "Need to consider other transit options: springball, HBJ."
+      "clearsObstacles": ["B"],
+      "note": "Grappling the block does not require canPreciseGrapple if Mama Turtle is asleep, as there is no risk of death on a miss."
     },
     {
-      "link": [6, 3],
-      "name": "Base",
+      "link": [6, 2],
+      "name": "Tricky Speed HiJump",
       "requires": [
-        {"or": [
-          "canHeroShot",
-          "h_canUsePowerBombs",
-          "Spazer",
-          "Wave"
-        ]}
+        "HiJump",
+        "canTrickyJump",
+        "SpeedBooster",
+        "canWalljump"
+      ],
+      "clearsObstacles": ["B"],
+      "note": [
+        "Briefly run on the top ledge on either side (left is slightly easier) then jump and wall jump on the grapple block."
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Tricky Speed HiJump, Walljumpless",
```

Comment:
Should an angry turtle be considered?  You could come up with a cue using wall shakes to safely fall after collecting the item.  Or unmorphing on the way down would be safe.